tests/multigrid-global-coarsening/multigrid_p_01.cc

```diff
@@ -0,0 +1,161 @@
+          mapping = std::make_unique<MappingFE<dim>>(FE_Q<dim>(1));
+        }
+
+      if (l == max_level)
+        mapping_ = mapping->clone();
+
+      // set up dofhandler
+      dof_handler.distribute_dofs(*fe);
+
+      // set up constraints
+      IndexSet locally_relevant_dofs;
+      DoFTools::extract_locally_relevant_dofs(dof_handler,
+                                              locally_relevant_dofs);
+      constraint.reinit(locally_relevant_dofs);
+      VectorTools::interpolate_boundary_values(
+        *mapping, dof_handler, 0, Functions::ZeroFunction<dim>(), constraint);
+      constraint.close();
+
+      // set up operator
+      op.reinit(*mapping, dof_handler, *quad, constraint);
+    }
+
+  // set up transfer operator
+  for (unsigned int l = min_level; l < max_level; ++l)
+    transfers[l + 1].reinit_polynomial_transfer(dof_handlers[l + 1],
+                                                dof_handlers[l],
+                                                constraints[l + 1],
+                                                constraints[l]);
+
+  MGTransferGlobalCoarsening<Operator<dim, Number>, VectorType> transfer(
+    operators, transfers);
+
+  GMGParameters mg_data; // TODO
+
+  VectorType dst, src;
+  operators[max_level].initialize_dof_vector(dst);
+  operators[max_level].initialize_dof_vector(src);
+
+  operators[max_level].rhs(src);
+
+  ReductionControl solver_control(
+    mg_data.maxiter, mg_data.abstol, mg_data.reltol, false, false);
+
+  mg_solve(solver_control,
+           dst,
+           src,
+           mg_data,
+           dof_handlers[max_level],
+           operators[max_level],
+           operators,
+           transfer);
+
+  deallog << dim << " " << fe_degree_fine << " " << n_refinements << " "
+          << (do_simplex_mesh ? "tri " : "quad") << " "
+          << solver_control.last_step() << std::endl;
+
+  DataOut<dim> data_out;
+
+  data_out.attach_dof_handler(dof_handlers[max_level]);
+  data_out.add_data_vector(dst, "solution");
+  data_out.build_patches(*mapping_, 2);
+
+  static unsigned int counter = 0;
+  std::ofstream       output("test." + std::to_string(dim) + "." +
+                       std::to_string(counter++) + ".vtk");
+  data_out.write_vtk(output);
+}
+
+int
+main(int argc, char **argv)
+{
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
+  MPILogInitAll                    all;
+
+  deallog.precision(8);
+
+  for (unsigned int n_refinements = 2; n_refinements <= 4; ++n_refinements)
+    for (unsigned int degree = 2; degree <= 4; ++degree)
+      test<2>(n_refinements, degree, false /*quadrilateral*/);
+
+  return 0; // TODO: enable simplex test once MGTwoLevelTransfer works for
+            // simplex meshes
+
+  for (unsigned int n_refinements = 2; n_refinements <= 4; ++n_refinements)
+    for (unsigned int degree = 2; degree <= 2; ++degree)
+      test<2>(n_refinements, degree, true /*triangle*/);
```

Comment:
I will enable this in #11429. But nevertheless this PR can be merged from my side since it adds a long overdue test.